flink-read-write-hdfs/README.md

```diff
@@ -49,10 +49,10 @@ folder to run this example.
    $ python -m pip install --upgrade feathub-nightly
    ```
 
-2. Build the Flink image to support HDFS.
+2. Build the Docker image with Feathub Flink processor.
```

Comment:
The original statement seems more informative as it explains why we need to build this image instead of using the official Flink docker image.